src/parser/jobs/mch/Wildfire.js

```diff
@@ -9,6 +9,8 @@ import STATUSES from 'data/STATUSES'
 import Module from 'parser/core/Module'
 import {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
 
+const DEBUFF_APPLICATION_BUFFER = 1000 // Buffer for the empty window check, since debuff applications always happen 9 times in the logs for some ungodly reason
```

Comment:
They _what_?

Reply:
You haven't seen that? There'll be like 9 consecutive `applydebuff` events (don't remember offhand if the same happens for `applybuff`) every time. Usually they're on the same millisecond but I ran into a few while testing this where some would be 1ms later than the first ones, so I added in wiggle room to be safe.